### PR TITLE
Added wilba.tech WT65-H1

### PR DESCRIPTION
**Pull request by Wilba6582**
## Description

Added wilba.tech WT65-H1

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by drashna — APPROVED** [review]

File: keyboards/wilba_tech/wt65_h1/info.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+    "keyboard_name": "wilba.tech WT65-H1", 
+    "url": "https://wilba.tech", 
+    "maintainer": "Wilba", 
+    "width": 16, 
+    "height": 5, 
+    "layouts": {
+        "LAYOUT_all": {
```
Comment (noroadsleft) [inline keyboards/wilba_tech/wt65_h1/info.json:8]: This is one of our Community Layouts.

```suggestion
        "LAYOUT_65_ansi_blocker_tsangan": {
```

File: keyboards/wilba_tech/wt65_h1/keymaps/default/keymap.c [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+// Default layout for WT65-H1
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+	// Default layer
+	[0] = LAYOUT_all(
```
Comment (noroadsleft) [inline keyboards/wilba_tech/wt65_h1/keymaps/default/keymap.c:22]:
```suggestion
	[0] = LAYOUT_65_ansi_blocker_tsangan(
```

File: keyboards/wilba_tech/wt65_h1/keymaps/default/keymap.c [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+		KC_ESC,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSPC,          KC_HOME,
+		KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS,          KC_PGUP,
+		KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT, KC_ENT,                    KC_PGDN,
+		KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT,                   KC_UP,   KC_END,
+		KC_LCTL, KC_LGUI, KC_LALT,                            KC_SPC,                                      MO(1),            KC_LEFT, KC_DOWN, KC_RGHT),
+
+	// Fn1 Layer
+	[1] = LAYOUT_all(
```
Comment (noroadsleft) [inline keyboards/wilba_tech/wt65_h1/keymaps/default/keymap.c:30]:
```suggestion
	[1] = LAYOUT_65_ansi_blocker_tsangan(
```

File: keyboards/wilba_tech/wt65_h1/keymaps/default/keymap.c [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+		KC_GRV,  KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  KC_DEL,           KC_TRNS,
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_PSCR, KC_SLCK, KC_PAUS, KC_TRNS, KC_TRNS, KC_TRNS,          KC_TRNS,
+		KC_TRNS, KC_VOLD, KC_VOLU, KC_MUTE, KC_EJCT, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,                   KC_TRNS,
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,                   KC_TRNS, KC_TRNS,
+		KC_TRNS, KC_TRNS, KC_TRNS,                            KC_TRNS,                                     KC_TRNS,          KC_TRNS, KC_TRNS, KC_TRNS),
+
+	// Fn2 Layer
+	[2] = LAYOUT_all(
```
Comment (noroadsleft) [inline keyboards/wilba_tech/wt65_h1/keymaps/default/keymap.c:38]:
```suggestion
	[2] = LAYOUT_65_ansi_blocker_tsangan(
```

File: keyboards/wilba_tech/wt65_h1/keymaps/default/keymap.c [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,          KC_TRNS,
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,          KC_TRNS,
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,                   KC_TRNS,
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,                   KC_TRNS, KC_TRNS,
+		KC_TRNS, KC_TRNS, KC_TRNS,                            KC_TRNS,                                     KC_TRNS,          KC_TRNS, KC_TRNS, KC_TRNS),
+
+	// Fn3 Layer
+	[3] = LAYOUT_all(
```
Comment (noroadsleft) [inline keyboards/wilba_tech/wt65_h1/keymaps/default/keymap.c:46]:
```suggestion
	[3] = LAYOUT_65_ansi_blocker_tsangan(
```

File: keyboards/wilba_tech/wt65_h1/rules.mk [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = yes           # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+BLUETOOTH_ENABLE = no       # Enable Bluetooth
+AUDIO_ENABLE = no           # Audio output
```
Comment (noroadsleft) [inline keyboards/wilba_tech/wt65_h1/rules.mk:22]:
```suggestion
AUDIO_ENABLE = no           # Audio output

LAYOUTS = 65_ansi_blocker_tsangan
```

File: keyboards/wilba_tech/wt65_h1/wt65_h1.h [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "quantum.h"
+
+#define ____ KC_NO
+
+#define LAYOUT_all( \
```
Comment (noroadsleft) [inline keyboards/wilba_tech/wt65_h1/wt65_h1.h:23]:
```suggestion
// This is equivalent to LAYOUT_all
#define LAYOUT_65_ansi_blocker_tsangan( \
```

**Review by noroadsleft — CHANGES_REQUESTED** [review]

**Review by noroadsleft — APPROVED** [review]

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**